packages/@react-stately/color/src/useColorFieldState.ts

```diff
@@ -65,14 +74,19 @@ export function useColorFieldState(
   let {
     value,
     defaultValue,
-    onChange
+    onChange,
+    allowsAlpha = false
   } = props;
+  let hexFormat: ColorFormat = allowsAlpha ? 'hexa' : 'hex';
+  let validationRegex = allowsAlpha ? RGBA_REGEX : RGB_REGEX;
+  let minColor = allowsAlpha ? MIN_COLOR_ALPHA_DEFAULT : MIN_COLOR;
```

Comment:
I'm not super familiar with how ColorFields that support alpha usually work (XD/Photoshop/native color input all have each value as separate fields) but should the minColor here be `MIN_COLOR_ALPHA` instead? I noticed that the ColorField currently moves to `000000FF` when hitting fn + Left arrow on mac, but the user is able to modify the alpha channel to values smaller than "FF". To me "FF" makes for a perfectly reasonable alpha default but kinda weird that End doesn't actually go to the actual smallest value unless we are going for an approach where the arrow keys only modify the RGB values. If that is the case, it would be nice if the alpha channel didn't reset back to "FF" when shifting the values using the keyboard or maybe doing something like Datepicker where we have segments for each of the values but definitely out of scope for now haha.

Reply:
Clarification for `it would be nice if the alpha channel didn't reset back to "FF" when shifting the values using the keyboard`: looks like this only happens when hitting the min/max RGB value via arrow keys or Home/End. Might be an idea to preserve the alpha value at all times for all keyboard interactions?